.github/workflows/verify-org-config.yaml

```diff
@@ -1,16 +1,23 @@
 name: Verify Org Config
 on:
-  pull_request:
+  pull_request_target:
     branches:
       - master
 jobs:
   verify-org-config:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
+      - name: Checkout master
         uses: actions/checkout@v2.3.1
         with:
           persist-credentials: false
+          
+      - name: Checkout PR
```

Comment:
to be really exact: this is the branch, not the PR

Reply:
you cant checkout PRs, only branches

Reply:
```suggestion
      - name: Checkout PR branch
```